controllers/microk8sconfig_controller.go

```diff
@@ -396,6 +400,11 @@ func (r *MicroK8sConfigReconciler) handleJoiningControlPlaneNode(ctx context.Con
 		controlPlaneInput.HTTPSProxy = microk8sConfig.Spec.InitConfiguration.HTTPSProxy
 		controlPlaneInput.HTTPProxy = microk8sConfig.Spec.InitConfiguration.HTTPProxy
 		controlPlaneInput.NoProxy = microk8sConfig.Spec.InitConfiguration.NoProxy
+
+		if microk8sConfig.Spec.InitConfiguration.IPinIP {
+			controlPlaneInput.IPinIP = true
+		}
```

Comment:
```suggestion
		controlPlaneInput.IPinIP = microk8sConfig.Spec.InitConfiguration.IPinIP
```
any reason why not this?

Reply:
Done